documentation/src/main/asciidoc/reference/troubleshooting.asciidoc

```diff
@@ -0,0 +1,103 @@
+[[troubleshooting]]
+= Troubleshooting
+
+[[troubleshooting-under-the-hood]]
+== Finding out what is executed under the hood
+
+For search queries, you can get a human-readable representation of a <<search-dsl-query-object,`SearchQuery` object>>
+by calling `toString()` or `queryString()`.
+Alternatively, rely on the logs:
+<<troubleshooting-logging-query,`org.hibernate.search.query`>> will log every query at the `DEBUG` level
+before it is executed.
+
+For more general information about what is being executed, rely on loggers:
+
+* <<troubleshooting-logging-lucene-infostream,`org.hibernate.search.backend.lucene.infostream`>>
+for Lucene.
+* <<troubleshooting-logging-elasticsearch-request,`org.hibernate.search.elasticsearch.request`>>
+for Elasticsearch.
```

Comment:
I don't know if it makes sense to have this wrap up chapter if we have the same information just above.